src/components/Global/PageTitle.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <header class="page-title">
+    <h1>{{ pageTitle }}</h1>
+    <p v-if="pageDescription">{{ pageDescription }}</p>
+  </header>
+</template>
+
+<script>
+export default {
+  name: "PageTitle",
+  props: ["pageTitle", "pageDescription"]
+};
+</script>
+
+<style lang="scss" scoped>
+h1 {
+  font-weight: $display4-weight;
+  line-height: $display-line-height;
+}
+.page-title {
+  margin-bottom: 2rem;
+}
+.page-description {
+  max-width: 750px;
```

Comment:
Is this to handle character spacing limits? I think this will work, but wonder about future scaling. One option, although not perfect either is to set the width between 60ch and 72ch.